Source/WebCore/page/scrolling/ScrollingCoordinatorTypes.h

```diff
@@ -132,6 +133,11 @@ struct RequestedScrollData {
     }
 };
 
+struct RequestedKeyboardScrollData {
+    bool stopImmediately { false };
```

Comment:
Using a bool makes this hard to read at the call site. Better to add an enum. I'd also put it second (since cancel is the less common action).

Reply:
A good change would be to have an `action` enum:
```
enum class KeyboardScrollAction : uint8_t {
  StartAnimation,
  StopWithAnimation,
  StopImmediately
}
```